graylog2-web-interface/src/pages/UserTokensEditPage.jsx

```diff
@@ -31,16 +31,16 @@ const PageTitle = ({ fullName }: {fullName: ?string}) => (
   </>
 );
 
-const _loadTokens = (username, currentUser, setTokens) => {
-  if (isPermitted(currentUser?.permissions, [`users:tokenlist:${username}`])) {
-    UsersDomain.loadTokens(username).then(setTokens);
+const _loadTokens = (userId, currentUser, setTokens) => {
+  if (isPermitted(currentUser?.permissions, [`users:tokenlist:${userId}`])) {
```

Comment:
for permission we need to use the `username`

Reply:
Good catch